### PR TITLE
Add test scenarios for privileged_commands_sudo

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add tests for `audit_rules_privileged_commands_sudo`

#### Rationale:

- There were only tests for "combined" rule `audit_rules_privileged_commands`

### CODE REVIEW

File: tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_4294967295_configured.pass.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# remediation = bash
+
+echo "-a always,exit -F path=/usr/bin/sudo -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged" >> /etc/audit/rules.d/privileged.rules
+# This is a trick to fail setup of this test in rhel6 systems
+ls /usr/lib/systemd/system/auditd.service
```
Comment (mildass) [inline tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_4294967295_configured.pass.sh]: Do we want this in `*.pass.sh` test?

Comment (dahaic) [reply tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_4294967295_configured.pass.sh]: Failure to setup scenario is reported in different way than rest of the execution/test. So yes, we do.

File: tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_default.fail.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# remediation = bash
+
+# augenrules is default for rhel7
+# This is a trick to fail setup of this test in rhel6 systems
+ls  /usr/lib/systemd/system/auditd.service
```
Comment (dahaic) [inline tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_default.fail.sh]: I know I myself introduced some "default" tests. But it has no value to actually run such scenario. Please remove

**Comment by dahaic** [comment]
Thank you @yuumasato . Good stuff.

**Pull request merged**